nuntium/urls.py

```diff
@@ -10,6 +34,27 @@
 
     url(r'^search/?$', MessageSearchView(), name='search_messages'),
 
+    url(r'^confirm_message/(?P<slug>[-\w]+)/?$', ConfirmView.as_view(), name='confirm'),
+
+    url(r'^moderation_accept/(?P<slug>[-\w]+)/?$', AcceptModerationView.as_view(), name='moderation_accept'),
+    url(r'^moderation_reject/(?P<slug>[-\w]+)/?$', RejectModerationView.as_view(), name='moderation_rejected'),
+
+)
 
-    url(r'^accounts/login/$', login, name='account_login')
+managepatterns = patterns('',
```

Comment:
I just loved this, but I'm wondering if it makes sense to have this patterns over at user_section/urls.py? At some point I thought of the user_section module as the management module (of course everything is changeable).